list.md

```diff
@@ -7,175 +7,68 @@ librarians.
+| Germany     | Tübingen        | [Universitätsbibliothek Tübingen](https://github.com/ubtue)                                                           |              |
+| Germany     | Weimar          | [Bibliothek der Hochschule für Musik FRANZ LISZT Weimar](https://github.com/HfMWeimar)                                |              |
+| Germany     |                 | [Gemeinsamer Verbünde Index (GVI)](https://github.com/gemeinsamerverbuendeindex)                                      |              |
+| Germany     |                 | [Deutsche Digitale Bibliothek](https://github.com/Deutsche-Digitale-Bibliothek)                                       |              |
+| Germany     |                 | [Zeitschriftendatenbank (ZDB)](https://github.com/Zeitschriftendatenbank)                                             |              |
+| Germany     |                 | [DINI AG KIM](https://github.com/dini-ag-kim)                                                                         |              |
+| Germany     |                 | [BVB-KOBV-ALLIANZ](https://github.com/bvb-kobv-allianz)                                                               |              |
+| Germany     |                 | [Max Planck Digital Library - MPDL](https://github.com/MPDL)                                                          |              |
+| Germany     |                 | [FID Jüdische Studien](https://github.com/FID-Judaica)                                                                |              |
+| Germany     |                 | [Fachinformationsdienst Darstellende Kunst](https://github.com/FID-Darstellende-Kunst)                                |              |
+| Luxembourg  |                 | [National Library of Luxembourg](https://github.com/natliblux)                                                        |              |
+| Netherlands |                 | [National Library of the Netherlands / Research](https://github.com/KBNLresearch)                                     |              |
+| Norway      | Bergen          | [University of Bergen Library (Norway)](https://github.com/ubbdst)                                                    |              |
+| Norway      | Oslo            | [Digital Development Team at Oslo Public Library](https://github.com/digibib)                                         |              |
+| Norway      | Oslo            | [University of Oslo Library (Norway)](https://github.com/scriptotek)                                                  |              |
+| Norway      |                 | [NTNU University Library (Norway)](https://github.com/NTNUUB)                                                         |              |
+| Norway      |                 | [BI Norwegian Business School Library (Norway)](https://github.com/bisentralen)                                       |              |
```

Comment:
We can delete now the country in the parenthesis. Also for some of the other lines.